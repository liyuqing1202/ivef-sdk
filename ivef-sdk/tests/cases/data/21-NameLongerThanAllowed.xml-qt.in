<MSG_LoginRequest xmlns="urn:http://www.ivef.org/XMLSchema/IVEF/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:http://www.ivef.org/XMLSchema/IVEF/1.1
\\projsrvr\isis\develop\ivef-sdk\trunk\doc\ivef1.xsd">
	<Header MsgRefId="1224-5667-8901-1234" Version="1.2"/>
	<Body>
		<LoginRequest Encryption="1" Name="lotsOfCharactersThatGoWayBeyondTheAllowed256Characters__Whhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh" Password="Very Secret"/>
	</Body>
</MSG_LoginRequest>
