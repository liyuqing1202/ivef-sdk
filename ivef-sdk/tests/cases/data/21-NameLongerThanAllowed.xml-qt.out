TestApp ready for input
error set Name = lotsOfCharactersThatGoWayBeyondTheAllowed256Characters__Whhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh at line 5 (column 351)
MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 1.2
    Body
        LoginRequest
            Encryption = 1
            Name = 
            Password = Very Secret
Body:LoginRequest:Name not set
TestApp shutting down
